add comment controller file and routes

17oogway04 opened the pull request:
Add comment controller files with all the respective routes

Commit:
add comment controller file and routes

17oogway04 commented:
Its got the wrong branch name but thats just the one I made the changes in

PenelopeWedemeyer reviewed (changes requested):
There needs to be a GetComment method (gets a single comment by its id). Also, I do not know if you want to add this now, but you need to add the authorization to the create, update, and delete methods.
Another thing I noticed was that after the ActionResult you added the Comment (suppose to be within <> but that made it invisible). I am not sure if that is okay, but if it still works then there is no problem XD

17oogway04 committed:
Update CommentController.cs

added the authorization statements above the routes and added the using statements

17oogway04 committed:
Update CommentController.cs

PenelopeWedemeyer reviewed (approved):
Seems good!
